lib/ansible/modules/network/ios/ios_static_route.py

```diff
@@ -220,7 +236,6 @@ def main():
 
     module = AnsibleModule(argument_spec=argument_spec,
                            required_one_of=required_one_of,
-                           required_together=required_together,
```

Comment:
Why `required_together` is removed from here. Since either `prefix` or `aggregate` will be present this check will handle the `prefix` `required_together` check at the top level. For aggregate case, `required_together` check can be added as.
```
    argument_spec = dict(
        aggregate=dict(type='list', elements='dict', options=aggregate_spec, required_together=required_together),
    )
```

Reply:
We can't use `required_tegether` in the usual way, because some of those options might be specified only in `aggregate`. Instead, we run `_check_required_together` either once we've put together the full param list from aggregate, or when we determine that aggregate is not being used. If you've got an example that works without all this, I'd love to make this more straightforward.